das_client/app/lib/repository/das_config_repository_impl.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:app/api/backend_api_service.dart';
+import 'package:app/api/dto/logging_setting_dto.dart';
+import 'package:app/data/local/das_database_service.dart';
+import 'package:app/repository/das_config_repository.dart';
+import 'package:logging/logging.dart';
+
+final _log = Logger('DasConfigRepositoryImpl');
+
+class DasConfigRepositoryImpl implements DasConfigRepository {
+  DasConfigRepositoryImpl({required this.apiService, required this.databaseService}) {
+    init();
+  }
+
+  final BackendApiService apiService;
+  final DASDatabaseService databaseService;
+
+  LoggingSettingDto? _loggingSetting;
+
+  void init() async {
+    final success = await loadSettings();
+    if (!success) {
+      Future.delayed(Duration(minutes: 1)).then((value) {
```

Comment:
Nitpick: Sync-Zeit in statische Variable oder konfigurierbar? `Duration(minutes: 1)`